tests/FakeItEasy.Specs/DummyCreationSpecs.cs

```diff
@@ -455,7 +455,7 @@ public void TupleCreation(Tuple<Foo, string> dummy)
         }
 
         [Scenario]
-        public void ValueTupleCreation((Foo foo, string s) dummy)
+        public void ValueTupleCreation((Foo Foo, string S) dummy)

```

Comment:
Did StyleCop give a warning for this? It's annoying... I tend to prefer camelCase name for tuple members. If you agree, we can probably configure the rule.

Reply:
It did warn. I have no strong opinion. Will reconfigure for camelCase.